### PR TITLE
getHashEquivalencePod should return if equivalence pod is found

### PR DESCRIPTION
**Pull request by sakeven**
Signed-off-by: sakeven <jc5930@sina.cn>

**What this PR does / why we need it**:

getHashEquivalencePod should return if equivalence pod is found, rather than simply check equivalenceHash equals to 0 later.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by bsalamat** [comment]
@sakeven Why do we need this change? Is it only a readability improvement?

**Comment by sakeven** [comment]
@bsalamat  Readability improvement is one point. I don't know if it's possible that equivalencePod exists  while its hash is 0.

**Comment by bsalamat** [comment]
@sakeven equivalence cache uses [FNV-1a hash](https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function) which creates a non-zero hash AFAIK.

**Comment by bsalamat** [comment]
I was wrong about the fact that the hash could never be zero.

/lgtm

**Comment by sakeven** [comment]
@jayunit100 PTAL.

File: plugin/pkg/scheduler/core/equivalence_cache_test.go [hunk 1]
```diff
@@ -305,11 +305,11 @@ func TestGetHashEquivalencePod(t *testing.T) {
...
 		},
 	}
-	hash4 := ecache.getHashEquivalencePod(pod4)
+	hash4, found := ecache.getHashEquivalencePod(pod4)
 
-	if hash4 != 0 {
-		t.Errorf("Failed: equivalence hash of pod %v is expected to be: 0, but got: %v",
-			pod4.Name, hash4)
+	if found {
+		t.Errorf("Failed: equivalence hash of pod %v is expected to not found, but got: %v",
```
Comment (bsalamat) [inline plugin/pkg/scheduler/core/equivalence_cache_test.go]: nit:s/is expected to not found/is not expected to be found

**Review by bsalamat — COMMENTED** [review]
Thanks, looks good. Just a minor nit.

**Comment by sakeven** [comment]
Any update?

**Comment by bsalamat** [comment]
@k82cn could you please approve this PR?

**Comment by k82cn** [comment]
sorry for the delay, I'll review this today.

**Pull request merged**